lib/build_manager.py

```diff
@@ -46,5 +48,11 @@ def __call__(self):
         self.build()
 
     def build(self):
-        scheduler = lib.scheduler.Scheduler()
-        self.distro.build_packages(scheduler(self.packages_manager.packages))
+        packages_order = self.packages_manager.packages
+        if CONF.get("default").get("build_dependencies"):
+            scheduler = lib.scheduler.Scheduler()
+            packages_order = scheduler(packages_order)
+        else:
+            LOG.info("Not scheduling build of dependencies")
+            LOG.info("Building packages: {}".format(packages_order))
```

Comment:
Flexibility is always nice, but yeah, I don't see any good reason to do this either.